src/Package/Truck.java

```diff
@@ -1,19 +1,19 @@
 package Package;
 
-public class Truck {
 
-    public String modelName;
-    public int wheelsCount;
-
-    public void updateTyre() {
-        System.out.println("Меняем покрышку");
+public class Truck extends Vehicle{
+    public Truck(String modelName, int wheelsCount) {
+        super(modelName, wheelsCount);
     }
-
     public void checkEngine() {
         System.out.println("Проверяем двигатель");
```

Comment:
```suggestion
        System.out.println("Проверяем двигатель грузовика");
```
чтобы подчеркнуть, что реализации разные и есть полиморфизм